internal/core/algorithm/ngt/option.go

```diff
@@ -104,58 +115,69 @@ func WithDistanceTypeByString(dt string) Option {
 		d = Hamming
 	case "cosine", "cos":
 		d = Cosine
-	case "normalizedangle", "normalized angle", "normalized ang", "nang", "nangle":
+	case "normalizedangle", "normalizedang", "nang", "nangle":
 		d = NormalizedAngle
-	case "normalizedcosine", "normalized cosine", "normalized cos", "ncos", "ncosine":
+	case "normalizedcosine", "normalizedcos", "ncos", "ncosine":
```

Comment:
Could you please teach me about these changes? 🙏

Reply:
due to the logic in line 107, the space character will be removed, so comparing with string contains space is no sense, so I removed it.

Reply:
Did you add the test case about your changes?

Reply:
Sorry I didn't add the test case for it. I will add new test case for it later.